src/saving/NewSaveMenu.cs

```diff
@@ -167,4 +167,13 @@ private void OnSaveNameTextEntered(string newName)
             ToolTipManager.Instance.ShowPopup(TranslationServer.Translate("INVALID_SAVE_NAME_POPUP"), 2.5f);
         }
     }
+
+    private void OnSaveListConfirmed(SaveListItem item)
+    {
+        saveNameBox.Text = item.SaveName;
+        overwriteConfirm.DialogText = string.Format(CultureInfo.CurrentCulture,
+            TranslationServer.Translate("THE_CHOSEN_FILENAME_ALREADY_EXISTS"),
+            item.SaveName);
+        overwriteConfirm.PopupCenteredShrink();
```

Comment:
Isn't this duplicate code? I think these should be placed into a common method that both places can use.